cmd/metal-api/internal/service/v1/size.go

```diff
@@ -50,7 +50,14 @@ type SizeReservationResponse struct {
 	Reservations       int               `json:"reservations" description:"the amount of reservations of this size reservation"`
 	UsedReservations   int               `json:"usedreservations" description:"the used amount of reservations of this size reservation"`
 	ProjectAllocations int               `json:"projectallocations" description:"the amount of allocations of this project referenced by this size reservation"`
-	Labels             map[string]string `json:"labels" description:"free labels associated with this size reservation."`
+	Labels             map[string]string `json:"labels,omitempty" description:"free labels associated with this size reservation."`
+}
+
+type SizeReservationListRequest struct {
+	SizeID      *string `json:"sizeid,omitempty" description:"the size id of this size reservation"`
+	Tenant      *string `json:"tenant,omitempty" description:"the tenant of this size reservation"`
+	ProjectID   *string `json:"projectid,omitempty" description:"the project id of this size reservation"`
+	PartitionID *string `json:"partitionid,omitempty" description:"the partition id of this size reservation"`
```

Comment:
Here its a different from the SizeSearchQuery ?

Reply:
Because I wanted to add the tenant, which is an information that the database layer is not aware of. :sweat: